package.json

```diff
@@ -8,9 +8,10 @@
   "author": "IBM Corp.",
   "main": "index.js",
   "scripts": {
+    "coverage": "nyc report --reporter=text-lcov | coveralls",
```

Comment:
I think you need to add `coveralls` to `devDependencies` too.

Reply:
See https://travis-ci.org/strongloop/loopback-connector-mongodb/jobs/547095364#L3364

```
$ npm run coverage
> loopback-connector-mongodb@4.2.0 coverage /home/travis/build/strongloop/loopback-connector-mongodb
> nyc report --reporter=text-lcov | coveralls
sh: 1: coveralls: not found
events.js:183
      throw er; // Unhandled 'error' event
      ^
Error: write EPIPE
    at WriteWrap.afterWrite [as oncomplete] (net.js:868:14)
```

Reply:
Thanks